src/components/regionalization/RegionalizationModal/RegionalizationModal.tsx

```diff
@@ -1,9 +1,9 @@
 import { Modal as UIModal } from '@faststore/ui'
+import Link from 'src/components/ui/Link'
+import Icon from 'src/components/ui/Icon'
 import { useEffect } from 'react'
-import PostalCodeInput from 'src/components/common/PostalCode'
+import PostalCodeInput from 'src/components/common/PostalCode/PostalCodeInput'
```

Comment:
If there's an `index` in the `PostalCode` folder, components should be imported from there:
```suggestion
import { PostalCodeInput } from 'src/components/common/PostalCode'
```